website/public/js/services/questServices.js

```diff
@@ -84,12 +85,22 @@
       $rootScope.selectedQuest = undefined;
     }
 
+    function questInit(){
+      Analytics.track({'hitType':'event','eventCategory':'behavior','eventAction':'quest','owner':true,'response':'accept','questName':$rootScope.selectedQuest.key});
+      $rootScope.party.$questAccept({key:$rootScope.selectedQuest.key}, function(){
+        $rootScope.party.$get();
+      });
+      $rootScope.$state.go('options.social.party');
```

Comment:
Inviting the party to your quest had a curious lack of user feedback; I added this so we jump to the Party page. Does that look right?